src/dodal/devices/i24/beamstop.py

```diff
@@ -0,0 +1,37 @@
+from enum import Enum
+
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.motion import Motor
+from ophyd_async.epics.signal import epics_signal_rw
+
+
+class BSPositions(str, Enum):
+    CHECKBEAM = "CheckBeam"
+    DATACOLLECTION = "Data Collection"
+    DATACOLLECTIONFAR = "Data Collection Far"
+    TRAYMOUNT = "Tray Mount"
```

Comment:
Nit: Would be good to have these hyphen separated e.g. `DATA_COLLECTION`, makes them a bit more readable